CONTRIBUTING.rst

```diff
@@ -10,18 +10,46 @@ Dimagi tracks most issues internally, but we use github's `issue tracker`_
 for public facing issues.  Feel free to browse the issues there and tackle
 any you feel equipped to do.  When you update or add comments to an issue
 please mention **@dimagiupdate** to send an alert to our internal issue
-tracking system.  Before submitting a PR, review our `Code Contributions and Review`_.  You may also be interested in our `CommCare Developers`_
-email group if you have questions or need feedback.
+tracking system.  Before submitting a PR, review our
+`Code Contributions and Review`_.  You may also be interested in the
+`Developers category`_ of the `CommCare Forum`_ if you have questions or
+need feedback.
 
 Bug Reports
 -----------
 To file a bug report, please follow the system outlined in our `bug
 reports`_ wiki page.  You are also welcome to submit an accompanying pull
 request if you are able to resolve the issue.
 
+Documentation Contributions
+---------------------------
+CommCare HQ's technical documentation is available at `Read the Docs`_.
+It is written in reStructuredText_.
+
+When documenting features, it is good practice to keep the documentation
+close to its code, so that it can easily be found. A good example of
+this is the documentation for User Configurable Reporting. The code is
+situated in the *corehq/apps/userreports/* directory, and its documentation
+is in *corehq/apps/userreports/README.rst*.
```

Comment:
Is it worth explicitly saying that `corehq/apps/myapp/README.rst` is typically a good starting point for documentation?